hrui/tests/phpunit/CRM/HRUI/HelperTest.php

```diff
@@ -12,36 +14,57 @@ class CRM_HRUI_HelperTest extends \PHPUnit_Framework_TestCase implements Headles
 
   use HRUITrait;
 
+  protected $requiredExtensions = [
+    'uk.co.compucorp.civicrm.hrcore', // required for fabricator
+    'org.civicrm.hrident', // creates the Identity custom group required by hrcase
+    'uk.co.compucorp.civicrm.tasksassignments', // if not enabled will try to redirect
+    'org.civicrm.hrcase' // creates the line manager relationship type
+  ];
+
+  /**
+   * @return CiviEnvBuilder
+   */
   public function setUpHeadless() {
-    // hrcase create ( Line Manager is ) relationship type which is need for the tests
     return \Civi\Test::headless()
+      ->install($this->requiredExtensions)
       ->installMe(__DIR__)
-      ->install('org.civicrm.hrcase')
       ->apply();
   }
 
+  /**
+   * Checks that the CRM_HRUI_Helper returns all line managers
+   */
   public function testGetLineManagersList() {
-    $contactParamsA = array("first_name" => "chrollo", "last_name" => "lucilfer");
-    $contactParamsB = array("first_name" => "hisoka", "last_name" => "morou");
-    $contactA = $this->createContact($contactParamsA);
-    $contactB = $this->createContact($contactParamsB)
-    ;
-    $this->createRelationship($contactA, $contactB, 'Line Manager is');
+    $relationshipType = 'Line Manager is';
+
+    $contactA = $this->createContact('chrollo', 'lucilfer');
```

Comment:
We should probably call the method this way:
`$this->createContact(['first_name'=>'chrollo', 'last_name' => 'lucilfer']);`
That way in case we need to fabricate with more parameters in the future we can do
`$this->createContact(['first_name'=>'chrollo', 'last_name' => 'lucilfer' 'nick_name' => 'BlaBla', 'is_deleted' => 0]);`